tao_compiler/mlir/disc/transforms/lower_tf.cc

```diff
@@ -1080,6 +1083,161 @@ LogicalResult convertQuantizedConv2DWithBiasAndRequantizeOp(func::FuncOp func) {
   return success();
 }
 
+// Returns 1D 64-bit dense elements attribute with the given values.
+static mlir::DenseIntElementsAttr GetI64ElementsAttr(
+    absl::Span<const int64_t> values, Builder* builder) {
```

Comment:
nit: better to use llvm::ArrayRef<int64_t>

Reply:
removed, use a existed function with `llvm::ArrayRef<int64_t>` input